qemu/tests/qemu_guest_agent.py

```diff
@@ -1580,14 +1605,11 @@ def gagent_check_file_read(self, test, params, env):
         # if guest os resource is enough, will return no error.
         # else it will return error like "insufficient system resource"
         # which is expected
-        try:
-            self.gagent.guest_file_read(ret_handle, count=1000000000)
-        except guest_agent.VAgentCmdError as detail:
-            info_insuffi = "Insufficient system resources exist to"
-            info_insuffi += " complete the requested service"
-            if not re.search(info_insuffi, str(detail)):
-                test.fail("Return error but is not the desired information: "
-                          "('%s')" % str(detail))
+        if params.get("os_type") == "linux" and main_qga_ver < 5:
+            _read_guest_file_with_count(1000000000)
+        else:
+            # For linux,count=1000000000 if main_qga_ver<5 else 10000000
```

Comment:
What about Windows? Directly `10000000`? Cause previously it is `1000000000`, you sure you want to reduce it?

Reply:
> What about Windows? Directly `10000000`? Cause previously it is `1000000000`, you sure you want to reduce it?

windows should be there as a condition as well , editing it .

Reply:
@vivianQizhu